api/tendrl/user_get.rst

```diff
@@ -1,2 +1,44 @@
 user_get
-*******************************************************************************
\ No newline at end of file
+*******************************************************************************
+
+:authors: 
+          - fbalak@redhat.com
+
+Description
+===========
+
+Check validity of responses from `List Users`_ and `Get Single User`_ API calls.
+
+Setup
+=====
+
+#. Create user via `valid_new_user` fixture populated with user data specified
```

Comment:
Please describe what user should be created. I think it is no good to link specific fixtures or code directly.